flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/api/TableEnvironmentTest.java

```diff
@@ -212,4 +336,39 @@ private static void assertTemporaryCreateTableFromDescriptor(
         assertThat(catalogTable.getOptions().get("connector")).isEqualTo("fake");
         assertThat(catalogTable.getOptions().get("a")).isEqualTo("Test");
     }
+
+    private static void assertCreateModelFromDescriptor(
+            TableEnvironmentMock tEnv, boolean ignoreIfExists) throws ModelNotExistException {
+        final String catalog = tEnv.getCurrentCatalog();
+        final String database = tEnv.getCurrentDatabase();
+
+        if (ignoreIfExists) {
+            tEnv.createModel("M", TEST_MODEL_DESCRIPTOR, true);
```

Comment:
if we extract this kind of assert into a separate method I would expect it working with different model names, not only `M`